backend/services/coworking/reservation.py

```diff
@@ -555,6 +590,13 @@ def draft_reservation(
         # Enforce request range is within bounds of walkin vs. pre-reserved policies
         bounds = TimeRange(start=start, end=end)
 
+        # Check if user has exceeded reservation limit
+        if request.room.id is not "SN156":
```

Comment:
Might be good to eventually move `"SN156"` into a constant or into the coworking policy.

Reply:
This was an error on my part. If I'm not mistaken, the CSXL gets a room id of None when people are making walkin reservations. Here is the permalink to where I believe that is happening:
https://github.com/unc-csxl/csxl.unc.edu/blob/4c0a16c179fb1b461a39276badbbcf3eaefa5daf/backend/services/coworking/reservation.py#L666
I would be willing to make that change for all reservations, but if that is something that you guys want done I think I should do it on a different branch.